en/sensor/leddar_one.md

```diff
@@ -1,44 +1,15 @@
 # LeddarOne Lidar
```

Comment:
@bkueng This is updated LeddarOne page. I've just done this one as a tester for you to review. My assumption is that all you need to do is set up the params as in your serial config doc - there is no longer any specific ports that need to use on each board. 

However I assume you STILL need to check the driver is present. Would we be better off saying it isn't in FMUv2 but can be expected in other boards?